common/src/main/resources/db/changelog/v001/AB2D-291-create-initial-tables.sql

```diff
@@ -0,0 +1,197 @@
+    email               VARCHAR(255)        NOT NULL,
+    sponsor_id          BIGINT              NOT NULL,
+    enabled             BOOLEAN             NOT NULL
+);
+
+ALTER TABLE user_account ADD CONSTRAINT "pk_user_account" PRIMARY KEY (id);
+ALTER TABLE user_account ADD CONSTRAINT "uc_user_account_user_name" UNIQUE (user_name);
+ALTER TABLE user_account ADD CONSTRAINT "uc_user_account_email" UNIQUE (email);
+ALTER TABLE user_account ADD CONSTRAINT "fk_user_account_to_sponsor"  FOREIGN KEY (sponsor_id) REFERENCES sponsor (id);
+
+
+--rollback DROP TABLE user_account;
+--  -------------------------------------------------------------------------------------------------------------------
+
+
+--changeset spathiyil:AB2D-291-CreateTable-role failOnError:true dbms:postgresql
+CREATE TABLE role
+(
+    id                  BIGSERIAL,
+    name                VARCHAR(64)         NOT NULL
+);
+
+ALTER TABLE role ADD CONSTRAINT "pk_role" PRIMARY KEY (id);
+ALTER TABLE role ADD CONSTRAINT "uc_role_name" UNIQUE (name);
+
+
+--rollback DROP TABLE role;
+--  -------------------------------------------------------------------------------------------------------------------
+
+
+--changeset spathiyil:AB2D-291-CreateTable-user_role failOnError:true dbms:postgresql
+CREATE TABLE user_role
+(
+    user_account_id     BIGINT              NOT NULL,
+    role_id             BIGINT              NOT NULL
+);
+
+ALTER TABLE user_role ADD CONSTRAINT "fk_user_role_to_user_account" FOREIGN KEY (user_account_id) REFERENCES user_account (id);
+ALTER TABLE user_role ADD CONSTRAINT "fk_user_role_to_role" FOREIGN KEY (role_id) REFERENCES role (id);
+ALTER TABLE user_role ADD CONSTRAINT "uc_user_role_user_account_id_role_id" UNIQUE (user_account_id, role_id);
+
+
+--rollback DROP TABLE user_role;
+--  -------------------------------------------------------------------------------------------------------------------
+
+
+--changeset spathiyil:AB2D-291-CreateTable-job_status failOnError:true dbms:postgresql
+CREATE TABLE job_status
+(
+    id                  BIGSERIAL,
+    name                VARCHAR(64)        	NOT NULL
+);
+
+ALTER TABLE job_status ADD CONSTRAINT "pk_job_status" PRIMARY KEY (id);
+ALTER TABLE job_status ADD CONSTRAINT "uc_job_status_name" UNIQUE (name);
+
+
+--rollback DROP TABLE job_status;
+--  -------------------------------------------------------------------------------------------------------------------
+
+
+--changeset spathiyil:AB2D-291-CreateTable-job failOnError:true dbms:postgresql
+CREATE TABLE job
+(
+    id                  BIGSERIAL,
+    job_id              VARCHAR(255)        NOT NULL,
+    user_account_id     BIGINT              NOT NULL,
+    created_at          TIMESTAMPTZ         NOT NULL,
+    expires_at          TIMESTAMPTZ         NOT NULL,
+    resource_types      VARCHAR(255)        NOT NULL,
+--  job_status_id       BIGINT              NOT NULL,
+    status              INT                 NOT NULL,
```

Comment:
Let's change to `varchar` and use `@Enumerated(EnumType.STRING)` on the corresponding entity's field. Using enum names instead of integer ordinals would be more readable (at a small cost of increased storage)

Reply:
done